src/protocol/cmds.rs

```diff
@@ -132,3 +133,32 @@ impl InvitedAnyCmdReq {
         })
     }
 }
+
+#[derive(Clone)]
+#[pyclass]
+pub(crate) struct AnonymousAnyCmdReq(pub anonymous_cmds::AnyCmdReq);
+
+#[pymethods]
+impl AnonymousAnyCmdReq {
+    fn dump<'py>(&self, python: Python<'py>) -> PyResult<&'py PyBytes> {
+        Ok(PyBytes::new(
+            python,
+            &self
+                .0
+                .dump()
+                .map_err(|e| ProtocolError::new_err(format!("encoding error: {e}")))?,
+        ))
+    }
+
+    #[classmethod]
+    fn load(_cls: &PyType, buf: Vec<u8>, py: Python) -> PyResult<PyObject> {
+        use anonymous_cmds::AnyCmdReq;
+
+        let cmd = AnyCmdReq::load(&buf)
```

Comment:
```suggestion
    fn load(_cls: &PyType, buf: &[u8], py: Python) -> PyResult<PyObject> {
        use anonymous_cmds::AnyCmdReq;

        let cmd = AnyCmdReq::load(buf)
```

Can we take a slice instead of coping the bytearray ?

Reply:
Good catch